### ISSUE TITLE
FBDoom graphics renders incorrectly if visible and virtual resolution is not matched (possible bug in fbDoom?)

### ISSUE THREAD

**Issue by shadlyd15** [shadlyd15/DoomLinux#6]
FBDoom graphics renders incorrectly if visible and virtual resolution is not matched.
@rnsi Did the "Linux logo" load correctly on Dell i5 Gen2 laptop with Intel graphics? Or was it scrambled as well? Can you please tell me the visible and virtual resolution? It is possibly a bug in fbDoom. Need to create patch for that.

**Comment by rnsi** [comment]
No, there's no Linux logo.  It's all text until fbdoom loads. Then fbdoom is overlaid on the text. There's 5 1/2 lines of text above and 4 1/2 lines of text below. fbdoom does not cover the entire screen.

**Comment by shadlyd15** [comment]
> No, there's no Linux logo. It's all text until fbdoom loads. Then fbdoom is overlaid on the text. There's 5 1/2 lines of text above and 4 1/2 lines of text below. fbdoom does not cover the entire screen.

Can you please exit from fbdoom with backspace key and check x_res,  x_virtual_res, y_res, y_virtual_res ?
`fbdoom -iwad /bin/doom1.wad > dump`
`cat dump`
